packaging/fedora/snapd.spec

```diff
@@ -380,12 +380,11 @@ autoreconf --force --install --verbose
 popd
 
 # Build systemd units
-pushd ./data/systemd
-make BINDIR="%{_bindir}" LIBEXECDIR="%{_libexecdir}" \
+make -C data \
```

Comment:
Please refrain from making unrelated changes to the spec. Changing from `pushd` / `popd` to using `make -C` is a different type of change.

Reply:
Sorry, I just inherited this change, I can revert this back if you prefer.